content/tutorials/unityxrpk-networked.mdx

```diff
@@ -0,0 +1,58 @@
+
+Before we start, we recommend you acquaint yourself with [Photon/PUN fundamentals](https://doc.photonengine.com/zh-cn/pun/v2/demos-and-tutorials/pun-basics-tutorial/intro).
+It's important to note this is just an example networking implementation - the XR Package-critical step will be described in step 2.
+
+## 0. Preparation
+
+As mentioned, you'll have to add your own App ID to Photon - this is a unique identifier for your app, and is used to identify your app when connecting to a specific Photon server. It's free to get (see above link), and once you have it, you'll need to add it to the project.
+Go to `Window/Photon Unity Networking/Highlight Server Settings"` and add your App ID. You should be good to go - hitting `Play` will demonstrate the stages of connecting to a server in the console and instantiate the Cubes prefab.
+
+## 1. General Scene Changes
+
+a. Note that the Left Controller and Right Controller scene objects now have additional components added to them: a Photon View and a Photon Transform View.
+These are built-in compoenents that, without any code, let us sync the position and rotation of the controller with the position and rotation of each controller in the Unity scene (assuming we're networked - which we'll discuss in the next step).
+
+![](./unity_xrpk_networked/1.png)
+
+It's important to note that even though were are not displaying the controllers graphically (since, when working with multi apps, we can see the controllers represented in our runtime environment like SteamVR), we are still able to sync the position and rotation of the controllers - furthermore this is critical to ensure that the cubes respond to each controller across _every other networked instance of the scene_, and not just in ours.
+
+Aside from those Photon components, the controller object is identical to the example in the previous tutorial.
+
+b. Next, take a look at the `Networking Manager` script under the `Managers` Scene object.
+The script is responsible for handling the connection to Photon. It is a variation on the basic connection script from the [Photon PUN Tutorial](https://doc.photonengine.com/zh-cn/pun/v2/demos-and-tutorials/pun-basics-tutorial/intro).
+The connection is handled automatically and straightforwardly, with one exception: the room id is procured from the Pluto _Multi App Manager_ - thus, ensuring a unique room is created for every Pluto conversation that the multi app is part of!
+
+## 2. More on the Multi App Manager
+
+Included in the Pluto Unity XRPK Exporter is a `MultiAppManager` static object that provides an entry point to several endpoints available to you when you launch your XRPK (although _not_ in the Unity editor!).
+One of them is access to the unique XR Package ID generated by the Pluto Multi App Launcher when it loads your XR Package.
```

Comment:
> One of them **has** access to the unique XR Package ID generated by the Pluto Multi App Launcher

Reply:
actually I think it's just "one of them is the...". Late night🙃